lib/html_tag.js

```diff
@@ -25,7 +25,7 @@ function htmlTag(tag, attrs, text, escape = true) {
     if (attrs[i] == null) result += '';
     else {
       if (i.match(regexUrl)
-        || (tag === 'meta' && !attrs[i].match(regexMeta) && Object.values(attrs)[0].match(regexMeta))) {
+        || (tag === 'meta' && !attrs[i].toString().match(regexMeta) && Object.values(attrs)[0].match(regexMeta))) {
```

Comment:
```suggestion
        || (tag === 'meta' && String(!attrs[i]).match(regexMeta) && Object.values(attrs)[0].match(regexMeta))) {
```

I prefer `String()` since only Object and Number have `toString()`.

Reply:
That's right. Thanks!!